src/app/Layout/PresentationSettings/PresentationSettings.tsx

```diff
@@ -5,32 +5,39 @@ import { PresentationType } from "../Types/PresentationType";
 
 export interface PresentationSettingsProps {
     currentPresentation: PresentationType;
+    districtSelected: string;
     displayedDecimals?: number;
     decimals: string;
     showPartiesWithoutSeats: boolean;
     changeDecimals: (decimals: string, decimalsNumber: number) => void;
     toggleShowPartiesWithoutSeats: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    selectDistrict: (event: React.ChangeEvent<HTMLSelectElement>) => void;
     results: LagueDhontResult;
 }
 export class PresentationSettings extends React.Component<PresentationSettingsProps> {
     static defaultProps = {} as PresentationSettingsProps;
-    onChange() {
```

Comment:
These changes were not strictly related to this PR either.

Reply:
I should definitely get better at using range reverts of files before staging.